app/lib/constants/site_config.rb

```diff
@@ -278,12 +266,8 @@ module SiteConfig
         description: "Add the secret key for Google reCAPTCHA, which is used for reporting abuse",
         placeholder: "What is the Google reCAPTCHA secret key?"
       },
-      right_navbar_svg_icon: {
-        description: "The SVG icon used to expand the right navbar navigation menu. Should be a max of 24x24px.",
-        placeholder: SVG_PLACEHOLDER
-      },
       secondary_logo_url: {
-        description: "Used as the secondary logo",
+        description: "If you have an alternate logo, this is used in some areas such as onboarding. It's typically the more\"playful\" version of the logo",
```

Comment:
```suggestion
        description: "A place for an alternate logo, if you have one. Used throughout member onboarding and in some sign in forms.",
```
It feels a bit odd for us to specify whether the logo is more playful or not here; I actually don't know if we should be doing that at all, but if that kind of guidance was going to live somewhere, I don't think it should be here (maybe in the creator onboarding guides or something like that?).